sci-mathematics/glucas/glucas-2.9.0.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="A client to make primality test for Mersenne numbers"
+DESCRIPTION="Glucas is related to GIMPS effort to search the biggest primes \
+ever found."
+HOMEPAGE="https://sourceforge.net/projects/glucas/"
+COPYRIGHT="2001 gbvalor"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/glucas/Glucas-$portVersion.tar.gz"
+CHECKSUM_SHA256="fde422220245788c0d8aeaec6a89f26e849905b1417c5e6ff1a01f56c595cadc"
+PATCHES="glucas-2.9.0.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	glucas = $portVersion
+	cmd:Glucas = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:gcc
+	cmd:make
+	"
+BUILD()
+{
+	autoreconf -fi
+	runConfigure ./configure
```

Comment:
Suggestion made by @fbrosson 
```diff
-	runConfigure ./configure
+	MAKEINFO=true runConfigure ./configure
```

Reply:
Actually, I noticed that if we don't add `cmd:makeinfo` then we won't get the `glucas.info` file.

Reply:
The `MAKEINFO=true` trick would have been ok if the `glucas.info` file was in the tarball. But it is not. It gets generated from a "texi" file.